docs/hugo/content/guide/crd-management.md

```diff
@@ -0,0 +1,121 @@
+---
+title: CRD management in ASO
+weight: -3
+---
+
+## How does CRD management work in Azure Service Operator?
+
+ASO performs CRD management differently than many traditional operators. You might notice if you install the
+[Helm chart](../installing-from-helm) or the [YAML](../installing-from-yaml) that there are NO CRDs installed by default.
+
+This is for two reasons:
+
+1. ASO supports so many CRDs that it exceeds the maximum Helm chart size when CRDs are included, so the CRDs cannot be 
+   included in the Helm chart.
+2. ASO supports so many CRDs that it can cause problems if they're all installed onto a Kubernetes cluster with a low
+   memory limit. See [#2920](https://github.com/Azure/azure-service-operator/issues/2920) for details.
+
+For the above reasons, by default each version of the ASO operator pod embeds the CRDs inside the pod and installs them when
+the pod launches. 
+
+**No CRDs are installed by default.** New CRDs will only be installed if they are specified in the 
+`--crd-pattern` cmdline argument to the `manager` container. `--crd-pattern` is a semicolon delimited string of CRD
+patterns to apply. 
+
+**Existing CRDs are always upgraded.** If a CRD already exists in the cluster (from a previous version of ASO or 
+a manual CRD installation), it will always be updated to the latest version of the CRD.
+
+## With Helm
+
+### Automatic CRD installation (recommended)
+
+Use the `crdPattern` value in `values.yaml` to specify what new CRDs to install. Existing CRDs will always be upgraded
+even if `crdPattern` is empty.
+
+Example: `resources.azure.com/*;containerservice.azure.com/*;keyvault.azure.com/*;managedidentity.azure.com/*;eventhub.azure.com/*`
+
+> **Note:** If you are specifying the Helm values from a shell with `--set crdPattern=...` you must wrap the crdPattern
+> argument in single quotes `'` if it contains wildcard characters the shell would expand. For example:
+> `--set crdPattern='resources.azure.com/*;containerservice.azure.com/*'`
+
+### Manual CRD installation
+
+If you do not want to grant the operator permission to create + update CRDs, you can instead manage the CRDs yourself.
+To do this you can use the customresourcedefinitions YAML that is part of each release. This YAML contains just the ASO CRDs.
+You can then filter to whichever subset of CRDs you want or apply them all. 
+See for example [azureserviceoperator_customresourcedefinitions_v2.0.0.yaml](https://github.com/Azure/azure-service-operator/releases/download/v2.0.0/azureserviceoperator_customresourcedefinitions_v2.0.0.yaml)
+
+The process to do this for a fresh install is:
+1. Install the CRDs you want to use. These must be the CRDs for the same version of the operator you install in step 2.
```

Comment:
I think, strictly, Markdown requires a blank line between paragraphs and lists.

Reply:
Fixed